docs/documentation/automation/automation_log.md

```diff
@@ -0,0 +1,162 @@
+import Highlight from '@theme/Highlight';
+
+## Overview {#overview}
+
+The _automation log_ helps you test and debug _routines_. Every time a routine or automation runs, its operation is recorded in the _automation log_. There you can find information concerning the automation's execution date, duration, owner, data context, and result. This tool allows you to see how your routines are working and do any necessary debugging.
+
+Whenever you test a routine (Run Routine) from the _Routine Builder_, the test will also be recorded in the log.
+
+Logs can be accessed directly through the _Administrative Panel_ or anywhere a routine can be built, like workflows, SLAs, or other automations.
+
+## Automation Log List {#log-list}
+
+<img alt="automation log" className="img_sizing item shadow--tl" src={useBaseUrl('img/automations_log_00.png')} />
+<br/>
+
+### List Details {#settings-panel}
+The initial settings panel displays the list of automations in descending chronological order. It is divided into the following four columns:
+
+- **Result**: Possible results are _executed_ or _failed_.
+- **Duration**: Time it takes for an automation to execute. Results return in seconds and milliseconds.
+- **Execution Date**: The date and time the automation executed. The date is in DD/MM/YY format.
+- **Options**: Viewing is the only option, but more alternatives are available once the log is [opened](#automation-log-details).
+
+:::note
+- Each automation will be stored in the log list for no more than seven days.
+- The automation log list refreshes every minute but can be manually refreshed by pressing the refresh icon found in the upper-right corner.
+:::
+
+### Log Details {#automation-log-details}
+Once an automation has been chosen for viewing, a new settings panel will appear and more information will be made available. 
+
+_The settings panel will look something like the following image:_
+
+<img alt="automation log" className="img_sizing item shadow--tl" src={useBaseUrl('img/automations_log_01.png')} />
+<br/>
+
+The information displayed in the settings panel is divided into the five sections –_log, schedule, context, routine, results_– explained below:
+
+<div className="container box">
+<div className="row table-row-1">
+<div className="col col--3"><b>Log:</b></div>
+<div className="col col--3">
+
+Displays the automation's basic information, which is also shown in the automation log [list](#settings-panel).
+
+</div>
+<div className="col col--6">
+
+- **Result**: Possible results are _executed_ or _failed_.
+- **Duration**: Time it takes for an automation to execute. Results return in seconds and milliseconds.
+- **Start & End Dates**: The date and time the automation executed. The date is in DD/MM/YY format.
+
+</div>
+</div>
+<div className="row table-row-2">
+<div className="col col--3"><b>Schedule:</b></div>
+<div className="col col--3">Displays links to the elements related to the routine.</div>
```

Comment:
"Displays links to the elements related to the routine" it can be confusing that is says routine again as a title,
Perhaps "Displays the programmed routine that was executed"

Reply:
nada que ver